counterparty-lib/counterpartylib/lib/api/routes.py

```diff
@@ -0,0 +1,255 @@
+        "args": [("status", 0)],
+    },
+    "/addresses/<address>/sweeps": {
+        "function": ledger.get_sweeps,
+    },
+    ### /assets ###
+    "/assets": {
+        "function": ledger.get_valid_assets,
+    },
+    "/assets/<asset>": {
+        "function": ledger.get_asset_info,
+    },
+    "/assets/<asset>/balances": {
+        "function": ledger.get_asset_balances,
+    },
+    "/assets/<asset>/balances/<address>": {
+        "function": ledger.get_balance_object,
+    },
+    "/assets/<asset>/orders": {
+        "function": ledger.get_orders_by_asset,
+        "args": [("status", "open")],
+    },
+    "/assets/<asset>/credits": {
+        "function": ledger.get_credits,
+    },
+    "/assets/<asset>/debits": {
+        "function": ledger.get_debits,
+    },
+    "/assets/<asset>/dividends": {
+        "function": ledger.get_dividends,
+    },
+    "/assets/<asset>/issuances": {
+        "function": ledger.get_issuances,
+    },
+    "/assets/<asset>/sends": {
+        "function": ledger.get_sends_or_receives,
+    },
+    "/assets/<asset>/dispensers": {
+        "function": ledger.get_dispensers,
+        "args": [("status", 0)],
+    },
+    "/assets/<asset>/dispensers/<address>": {
+        "function": ledger.get_dispensers,
+        "args": [("status", 0)],
+    },
+    "/assets/<asset>/holders": {
+        "function": ledger.get_asset_holders,
+    },
+    ### /orders ###
+    "/orders/<tx_hash>": {
+        "function": ledger.get_order,
+    },
+    "/orders/<tx_hash>/matches": {
+        "function": ledger.get_order_matches_by_order,
+        "args": [("status", "pending")],
+    },
+    "/orders/<tx_hash>/btcpays": {
+        "function": ledger.get_btcpays_by_order,
+        "args": [("status", "pending")],
+    },
+    ### /bets ###
+    "/bets/<tx_hash>": {
+        "function": ledger.get_bet,
+    },
+    "/bets/<tx_hash>/matches": {
+        "function": ledger.get_bet_matches_by_bet,
+        "args": [("status", "pending")],
+    },
+    "/bets/<tx_hash>/resolutions": {
+        "function": ledger.get_resolutions_by_bet,
+        "args": [("status", "pending")],
+    },
+    ### /burns ###
+    "/burns": {
+        "function": ledger.get_burns,
+    },
+    ### /dispensers ###
+    "/dispensers/<tx_hash>": {
+        "function": ledger.get_dispenser_info,
+    },
+    "/dispensers/<tx_hash>/dispenses": {
+        "function": ledger.get_dispenses,
+    },
+    ### /events ###
+    "/events": {
+        "function": ledger.get_events,
+        "args": [("last", None), ("limit", 100)],
+    },
+    "/events/<int:event_index>": {
+        "function": ledger.get_events,
+    },
+    "/events/counts": {
+        "function": ledger.get_events_counts,
+    },
+    "/events/<event>": {
+        "function": ledger.get_events,
+        "args": [("last", None), ("limit", 100)],
+    },
+    ### /healthz ###
+    "/healthz": {
+        "function": util.handle_healthz_route,
+        "args": [("check", "heavy")],
```

Comment:
should that be `check_type`?

Reply:
it could! Isn't a "heavy check" the same as a "heavy type check" ?

Reply:
ah no, it's the type of the check—heavy vs. light

a heavy `healthz` check is slower but tests more

Reply:
done